Removed remnants of pex_policy

srp3rd opened the pull request:
removed:

killcondor.py - used the old style pex_policy job description to identify and kill jobs.
writeNodeList.py - this was required in an early version of ctrl_orca to enumerate the nodes used (in pbs jobs, I believe).
ProvenanceRecorder.py - This was used to make database records for policy files used in a run.  This was in addition to the provenance information that used to be recorded via the provenance package, which is also obsolete, since it also used pex_policy.  I don't think a replacement was ever made for that package.

lsst/ctrl/orca/PolicyUtils.py - used to get all file names listed in a policy

Deleted line from ups/ctrl_orca.table, since pex_policy, is no longer required.

Removed "-r" option from bin.src/orca.py which referred to policy directory. This was an obsolete option.

srp3rd committed:
Removed remnants of pex_policy

removed:

killcondor.py - used the old style pex_policy job description to identify and kill jobs.
writeNodeList.py - this was required in an early version of ctrl_orca to enumerate the nodes used (in pbs jobs, I believe).
ProvenanceRecorder.py - This was used to make database records for policy files used in a run.  This was in addition to the provenance information that used to be recorded via the provenance package, which is also obsolete, since it also used pex_policy.  I don't think a replacement was ever made for that package.

lsst/ctrl/orca/PolicyUtils.py - used to get all file names listed in a policy

Deleted line from ups/ctrl_orca.table, since pex_policy, is no longer required.

Removed "-r" option from bin.src/orca.py which referred to policy directory. This was an obsolete option.

hsinfang commented:
Not really related to this PR, but it came up when I tried to use this ticket that eups cannot find the optional packages `condor` and `condor_glidein`.  I don't see such packages at github.com/lsst or lsst-dm either. Any ideas what they are?  Are they still needed?

srp3rd committed:
Remove "orca.repository" reference;  this defaulted to None, and defaults that way throughout the rest of
the code.  If we decide to use Config files repositories later, this can easily be re-added.

srp3rd committed:
Remove condor optional packages, since they aren't used.

srp3rd commented:
> Not really related to this PR, but it came up when I tried to use this ticket that eups cannot find the optional packages `condor` and `condor_glidein`. I don't see such packages at github.com/lsst or lsst-dm either. Any ideas what they are? Are they still needed?

One of the reasons we left it as optional when I asked K-T how to handle this (conversation was many years ago) was to indicate it has an external Condor dependency.    Since only this is only used by NCSA on  NCSA computers, we didn’t think having it as optional would be a big deal.   I removed it as optional.

hsinfang reviewed (approved):
LGTM
